businesstime/australia.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import date
+from businesstime import Holidays
+
+class QueenslandPublicHolidays(Holidays):
+    """
+    http://www.qld.gov.au/recreation/travel/holidays/public/
+
+    Current coverage is only 2013-2016.
+    """
+
+    # Australian public holidays are legislated by each state individually for 
```

Comment:
Yeah I am already going to be maintaining this list, I can just keep it in my script but I thought you might be interested in having it here as well.

We could try and forecast what future public holidays will be, but that might cause problems if the actual legislated holidays end up different than what this code was predicting, so I figured it's better not to like and just raise NotImplementedError if there is no hard data for the future.